Content.Server/Remotes/DoorRemoteSystem.cs

```diff
@@ -57,6 +56,7 @@ private void OnAfterInteract(EntityUid uid, DoorRemoteComponent component, After
                 || !TryComp<DoorComponent>(args.Target, out var doorComponent) // If it isn't a door we don't use it
                 || !HasComp<AccessReaderComponent>(args.Target) // Remotes do not work on doors without access requirements
                 || !TryComp<AirlockComponent>(args.Target, out var airlockComponent) // Remotes only work on airlocks
+                || !TryComp<SharedAirlockComponent>(args.Target, out var sharedAirlockComponent)
```

Comment:
You shouldn't ever need to get both the shared and server/client specific component. Just the server's  `AirlockComponent` should be enough here.

(there are some exceptions when a shared function that has a shared component as an argument gets overridden by the server, which might need the server component, but that's not the case here)

Reply:
Resolved by: https://github.com/space-wizards/space-station-14/pull/6500/commits/[37150b1](https://github.com/space-wizards/space-station-14/pull/6500/commits/37150b12956d477d4254930bcc73bd38d8a6c91e)2956d477d4254930bcc73bd38d8a6c91e